controller/backup_controller.go

```diff
@@ -716,18 +716,18 @@ func (bc *BackupController) syncBackupVolume(volumeName string) error {
 		// Request backup_volume_controller to reconcile BackupVolume immediately.
 		backupVolume.Spec.SyncRequestedAt = syncTime
 		if _, err = bc.ds.UpdateBackupVolume(backupVolume); err != nil && !apierrors.IsConflict(errors.Cause(err)) {
-			bc.logger.WithError(err).Errorf("Error updating backup volume %s spec", volumeName)
+			bc.logger.WithError(err).Errorf("Failed to update backup volume %v spec", volumeName)
```

Comment:
Why not error out here?

Reply:
I am also confused with this
In Backup/BackupVolume/BackupTarget controller, there are many place only log error but don't return 
some with comment 
```
// Ignore error to prevent enqueue
```

Reply:
Use `Warn` since it doesn't return err